package.json

```diff
@@ -21,9 +21,9 @@
     "build:test": "babel src/main/webapp/dev/ -x \".es6\" -d src/main/webapp/dev/",
     "build:src": "babel src/main/webapp/js/ -x \".es6\" -d src/main/webapp/js/",
     "build": "npm run build:src && npm run build:test",
-    "lint:src": "eslint src/main/webapp/js -c static-analysis/teammates-eslint-es6.yml --ext es6",
-    "lint:test": "eslint src/main/webapp/dev -c static-analysis/teammates-eslint-es6.yml --ext es6",
-    "lint:json": "eslint src/main/webapp/js/*.json src/main/resources/*.json src/test/resources/data/*.json -c static-analysis/teammates-eslint.yml",
+    "lint:src": "eslint src/main/webapp/js -c static-analysis/teammates-eslint.yml --ext .es6",
+    "lint:test": "eslint src/main/webapp/dev -c static-analysis/teammates-eslint.yml --ext .es6",
```

Comment:
No, it's not. I was wondering why we are using `es6` over `.es6` when the documentation mentioned the usage is to use with the period, do you happen to know?

In any case, I briefly looked at ESLint's [source code](https://github.com/eslint/eslint/blob/d21f52831fb1b2cc4f9632184788ab567e37625d/lib/util/glob-util.js#L45) and found that they replace the period and then add it later on, making the period optional.

I doubt that function will ever change so I do not think it will ever break. Or maybe they did that on purpose so in case people still missed the period it still worked.

Still, I think it should be changed because it will match the documentation that ESLint has and will not catch the attention of someone because it is different and not documented but works.